sympy/stats/error_prop.py

```diff
@@ -0,0 +1,44 @@
+"""Tools for arithmetic error propogation."""
+from itertools import repeat
+from sympy import Symbol, symbols, Add, Sum, Mul, simplify, Pow, exp
+
+def variance_prop(expr, consts=()):
+    """Symbolically propagates variance (sigma^2) for expressions.
+
+    Parameters
+    ==========
+    expr : Expr
+        A sympy expression to compute the variance for.
+    consts : sequence of Symbols, optional
+        Represents symbols that are known constants in the expr,
+        and thus have zero variance.
+
+    Returns
+    =======
+    var_expr : Expr
+        An expression for the total variance of the expr.
+        The variance for the original symbols (e.g. x) have a 'var_'
+        prepended to their name ('var_x').
+    """
+    args = expr.args
```

Comment:
@leosartaj - there is a warning in the docs saying [not to do this](http://docs.sympy.org/latest/tutorial/basic_operations.html#converting-strings-to-sympy-expressions) and it would be redundant on all the recursions.